File: backend/autotest/apis/system/statistics.py
# -*- coding: utf-8 -*-
# @author: xiaobai
from fastapi import APIRouter

from autotest.utils.response.http_response import partner_success
from autotest.services.system.statistic import StatisticService

router = APIRouter()


@router.post("/personalStatistics", description="个人统计")
async def personal_statistics():
    data = await StatisticService.personal_statistics()
    return partner_success(data)

File: frontend/src/api/useAutoApi/statistics.js
import request from '/@/utils/request';

/**
 * 统计
 */
export function usesStatisticsApi() {
  return {
    personalStatistics: (data) => {
      return request({
        url: '/statistics/personalStatistics',
        method: 'POST',
        data,
      });
    },
  };
}
